README.md

```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
 
 # Inputs and Outputs
 
-> TODO
+### Angular
+
+#### Parent Component
+```js
+@Component({
+  selector: 'parent',
+  template: `
+    <child
+      [user]="user"
+      (onEdit)="editedUser($event)"
+    >
+
+    </child>
+  `
+})
+
+export class ParentComponent {
+  user: User;
+  constructor() {
+
+    this.user = {
+      name: 'Foo Bar',
+      email: 'foobar@example.com'
+    }
+
+  }
+
+  editedUser(user: User){
+    console.log('Name of the edited user is', user.name);
+  }
+}
+
+```
+
+#### Child Component
+
+```js
+@Component({
+  selector: 'child',
+  template: `
+    <p>
+      <input type="text" value="{{user.name}}">
+    </p>
+    <p>
+      <input type="text" value="{{user.email}}">
+    </p>
```

Comment:
Could you remove this `<p>` with email? Just so it's more succinct, without extra clutter.